driver/esp32/ili9XXX.py

```diff
@@ -493,12 +506,11 @@ def stat(self):
         return time, setup, dma, px
 
 
-
 class ili9341(ili9XXX):
 
     def __init__(self,
         miso=5, mosi=18, clk=19, cs=13, dc=12, rst=4, power=14, backlight=15, backlight_on=0, power_on=0,
-        spihost=esp.HSPI_HOST, mhz=40, factor=4, hybrid=True, width=240, height=320,
+        spihost=esp.HSPI_HOST, mhz=40, factor=4, hybrid=True, width=240, height=320, start_x=0, start_y=0,
```

Comment:
If `start_x` and `start_y` are only relevant for `st7789` and never used on the other displays, does it make sense to receive them in each of the displays constructor?   

Instead you could simply not pass them on `super` so they would get their default 0 value.